### PR TITLE
Portfolio: finish desktop setup

### PR DESCRIPTION
**Pull request by rabayarabu**
Portfolio: finish desktop setup

1. Edit index.html file
2.  Edit style.css file
3.  Include images

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -6,651 +6,1075 @@
...
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-  height: 36px;
-}
+  .works {
+    display: grid;
+    justify-content: center;
+    align-content: center;
+  }
```
Comment (IngDavidTM) [inline style.css:121]: - [ ] You have done an excellent job, but it is crucial to ensure that your page matches the Figma template as closely as possible. As a result, there are a few issues that need to be addressed to enhance the appearance of your page. Specifically, I suggest making the following improvements:
- Centering the 'My Recent Works' section in the middle of your page would be more aesthetically pleasing.
- Currently, there seems to be a problem with the card rendering, as only two of them are displaying correctly. I suggest fixing this issue to ensure that all cards are displayed as intended, improving the overall look of your page.

**Your design**

Screenshot from 2023-04-05 08-16-12

Screenshot from 2023-04-05 08-16-30

**Figma**

Screenshot from 2023-04-05 08-20-52

Screenshot from 2023-04-05 08-21-15

File: style.css [hunk 2]
```diff
@@ -6,651 +6,1075 @@
...
+    background: #ff6b00;
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+    border: none;
+    font-weight: 700;
+    font-size: 17px;
+    letter-spacing: 0.03em;
+    color: #fff;
+  }
```
Comment (IngDavidTM) [inline style.css:443]: - [ ] Your work on the desktop version is outstanding! :+1: However, to proceed to the next stage of the project, there are a few issues that I kindly request you address. These issues are as follows:
- There is a border on your button that is not present in the Figma template. Kindly remove this border to ensure that the button matches the design accurately.
- The separator on your page needs to take up more space of the page width to match the design in the Figma template more closely. Making this adjustment will help ensure consistency in the design and improve the overall appearance of the page.

**Your design**

Screenshot from 2023-04-05 08-29-56

**Figma**

Screenshot from 2023-04-05 08-26-59

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @rabayarabu 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @rabayarabu, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well-implemented previous suggestions.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fixed bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**